cmds/report.c

```diff
@@ -194,7 +199,10 @@ static void print_function(struct uftrace_report_node *node, void *unused)
 		print_time_unit(node->total.sum);
 		pr_out("  ");
 		print_time_unit(node->self.sum);
-		pr_out("  %10"PRIu64 "  %-s\n", node->call, node->name);
+		pr_out("  %10"PRIu64 "  %-s", node->call, node->name);
+		if (node->loc)
```

Comment:
Thank you for finding and debugging this issue.
I intended task_find_sym_addr to return valid debug_location or NULL. So I added additional check to task_find_sym_addr function.

I found that some symbols described in .debug_info like below is not collected to *.dbg file though it seems like source location info. 
I'll try to know the rule when available, but if you can simply share the rule of collecting debug location from dwarf to *.dbg, It'll be helpful.
Thank you.
```
 6846068  <3><e89af4>: Abbrev Number: 398 (DW_TAG_subprogram)
 6846069     <e89af6>   DW_AT_name        : (indirect string, offset: 0x420b0a): PlatformWorkerThread 
 6846070     <e89afa>   DW_AT_decl_file   : 3
 6846071     <e89afb>   DW_AT_decl_line   : 29
 6846072     <e89afc>   DW_AT_declaration : 1
                                      // no DW_AT_linkage_name
```

Reply:
Sorry for late reply.  The basic rule is that it checks DW_TAG_subprogram DIE and reads decl_file and decl_line attributes.  It should have the location info if *.dbg file exists.

Reply:
@honggyukim The `task_find_loc_addr()` already checked the file pointer so I think it's ok.

Reply:
> Sorry for late reply. The basic rule is that it checks DW_TAG_subprogram DIE and reads decl_file and decl_line attributes. It should have the location info if *.dbg file exists.

Then do you think it would be an issue?

Reply:
No, I think it's ok now.